chat/README.md

```diff
@@ -0,0 +1,330 @@
+Chat App Demo
+=========================
+The `Chat` app demonstrates ([browse the source](https://github.com/revel/samples/tree/master/chat)):
```

Comment:
I'm not liking the "browser the source" links. Is there a better place for them?